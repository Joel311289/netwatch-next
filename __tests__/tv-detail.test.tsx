import { render } from '@testing-library/react';
import TVDetailPage from '@/pages/tv/[tvId]';

describe('Tests tv detail page', () => {
  test('renders page correctly', () => {
    const { container } = render(<TVDetailPage />);

    expect(container).toMatchSnapshot();
  });
});
