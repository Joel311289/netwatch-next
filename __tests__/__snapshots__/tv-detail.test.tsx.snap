// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests tv detail page renders page correctly 1`] = `
<div>
  <div
    class="wrapper hasMargins"
  />
</div>
`;
